core/commands/mfs.go

```diff
@@ -0,0 +1,819 @@
+package commands
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	gopath "path"
+	"strings"
+
+	key "github.com/ipfs/go-ipfs/blocks/key"
+	cmds "github.com/ipfs/go-ipfs/commands"
+	dag "github.com/ipfs/go-ipfs/merkledag"
+	mfs "github.com/ipfs/go-ipfs/mfs"
+	path "github.com/ipfs/go-ipfs/path"
+	ft "github.com/ipfs/go-ipfs/unixfs"
+
+	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+)
+
+type mfsMountListing struct {
+	Mounts []mfs.RootListing
+}
```

Comment:
can we scope all the mfs commands into its own dir + separate files (file per cmd)? this package is massive and it will help break things up later.